### ISSUE TITLE
How do we get support on this tool?

### ISSUE THREAD

**Issue by achaganti** [microsoft/nmarc#8]
Not getting the desiresd results especially with multiple value  variables

**Comment by brianly** [comment]
This is a personal endeavour that became an open source project to help some folks out understanding the alignment report. Unfortunately, I'm going to be out for an extended period soon, so I can't make commitments to major changes or support. The tool does expect C# experience for handling any exceptions, or customising the output.

It sounds like you've run into some issues that others have experienced though, and I've just made some updates to help with that. I'm going to update the readme with more detail on the outputs.

Taking a step back, I created this because I realised that a YAML file might be unwieldy in a lot of cases. Admins are much more familiar with CSV and that can be processed using Excel to create an action plan. I've provided this experience and feedback to the team responsible for the alignment report generator.  I understand they are investigating some improvements to the core reporting which should negate the need for this tool. I don't have an ETA, but I hope this will soon render this utility irrelevant and make things more convenient. If you don't have C# experience to jump in and make edits, I think waiting for a little bit for these updates may be the best option.
